README.md

```diff
@@ -4,7 +4,34 @@ In this repository, we collect example scripts that can be useful for the team a
 
 ## Prerequisites
 
-TBD
+To run python scripts inside `examples` directory you should first run the installer script:
+
+```bash
+$ bin/install
```

Comment:
imho this `bin/run` and `bin/install` are overkill. For the media downloaded I added the following and the users were happy with it, even though I forgot to include the `source venv/bin/activate` as a pre-requisite to run the script.

install: 

> - install a python 3 interpreter (if not installed already)
> - create a directory where you would run the script from.
> - copy the script to the directory.
> - install the ‘requests’ library by running ‘pip install -m requirements.txt’ from command line in that directory.